### DISCUSSION TITLE
Set MTU size when deploying LXC containers

### DISCUSSION THREAD

**Discussion by t0th0mas** [tteck/Proxmox#929] [Ideas]
Please add the possibility to set the MTU size in advanced configuration for LXC containers.

**Comment by tteck** [comment]
Looking for something like this?

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/dev/ct/debian-v5.sh)"
```

**Reply by t0th0mas** [reply]
Simple, but works. That's what I'm looking for :-)

**Reply by tteck** [reply]
Done!
